ui/scripts/ui-custom/instanceWizard.js

```diff
@@ -1359,18 +1359,18 @@
                     $wizard.find('div.data-disk-offering span.custom-disk-size').html(_s(old));
                 });
 
+                var wizardWidth = 950;
+                if($.browser.mozilla){
+                    wizardWidth = 896;
+                }
                 
                 var wizardDialog = $wizard.dialog({
                     title: _l('label.vm.add'),
-                    width: 896,
+                    width: wizardWidth,
```

Comment:
I'm still observing the same issue. It looks like you might have accidentally changed the width parameter instead of that of the height (as the height appears to be where the problem is).

Reply:
jQuery Ui is configuring the height to "auto". What is your resolution? Maybe that is why it is working here and not working at your computer?

Reply:
This is running in a VM on a MacBook Pro. The resolution seems to have been automatically configured to 1276x733.

Reply:
mine is 1333X768

Reply:
It's a bit difficult to extra from my MacBook what its exact resolution is. It likes you to choose Big, Default, Small sizes without telling you the details. On the Windows OS I'm running in a VM, though, it says it's 1276x733. I suspect my MacBook's display itself is more like your resolution.

Reply:
This is the picture from my chrome using the code of this PR:

Add zone:
![image](https://user-images.githubusercontent.com/4129005/43609348-1ca0c0aa-967a-11e8-8a0a-fffe30bccd57.png)

Add instance:
![image](https://user-images.githubusercontent.com/4129005/43609367-2c0145ec-967a-11e8-9d4d-8f0d5444d1bc.png)

I am not sure on what is the problem now.

Reply:
My chrome version is: 68.0.3440.84 (Official Build) (64-bit)

Reply:
Yeah, yours look perfectly fine.

I seem to have a problem on both Mac OS X (with Chrome and Safari) and Windows 7 (with Chrome, Firebox, and Explorer).

Reply:
¯\_(ツ)_/¯.....

I really have no clues now.

Reply:
I even changed my MacBook's resolution to the highest one and I still see the buttons hanging off the bottom of the window. Strange. Oh well, it's functional. Perhaps one of our GUI people can take a look at it later. Thanks!